common/archiver/s3store/mocks/generate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
question for the future: should we put copyright comments in .sh files? and enforce it with licensegen?